deployments/helm-chart/values.schema.json

```diff
@@ -1168,6 +1168,41 @@
             null
           ]
         },
+        "podDisruptionBudget": {
+          "type": "object",
+          "default": {},
+          "title": "The podDisruptionBudget Schema",
+          "required": [
+            "enabled"
+          ],
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+			"annotations": {
+              "type": "object",
+              "default": {},
+              "title": "The annotations Schema",
+              "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.25.4/_definitions.json#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta/properties/annotations"
+			},
+            "minAvailable": {
+              "type": "integer"
+            },
+            "maxUnavailable": {
+              "type": "integer"
+            }
+          },
+          "examples": [
+            {
+              "enable": true,
+              "minAvailable": 1,
+            }
```

Comment:
JSON is invalid. Missing comma `,` after this curly brace.

Reply:
Passes `cat values.schema.json | jq .`